pkgs/development/compilers/dasm/default.nix

```
@@ -0,0 +1,35 @@
+{stdenv, fetchFromGitHub}:
+
+stdenv.mkDerivation rec {
+	pname = "dasm";
+	version = "2.20.13";
+
+	src = fetchFromGitHub {
+		owner = "dasm-assembler";
+		repo = "dasm";
+		rev = "38c66afbb1773844c8e1bfedc1411d5863edcc49";
```

```suggestion
		rev = version;
```

We prefer to pin to the tagged release version.

---

Ah of course. Forgot I could refer to tags on github like that, Makes sense cus they're git refs. Thanks.